cset-workflow/meta/rose-meta.conf

```diff
@@ -115,697 +232,1353 @@ sort-key=met1
 ns=Environment
 description=Location of the conda virtual environment needed by METplus.
 type=quoted
+compulsory=true
 sort-key=met2
 
 [template variables=MET_INSTALL_DIR]
 ns=Environment
 description=Path where MET is installed to.
 type=quoted
+compulsory=true
 sort-key=met2
 
 [template variables=METPLUS_BASE]
 ns=Environment
 description=Path where METplus is installed to.
 type=quoted
+compulsory=true
 sort-key=met2
 
 [template variables=MET_LIBRARIES]
 ns=Environment
 description=Path to libraries needed by MET.
 type=quoted
+compulsory=true
 sort-key=met2
 
 
 ################################################################################
-# General
+# Models and Cases: Case and model settings.
 ################################################################################
 
-[template variables=LOGLEVEL]
-ns=General
-description=How detailed the logs should be.
-help=INFO only shows a general overview of what is happening, while DEBUG enables
-    in-operator logging, but produces a lot of spurious output.
-values="INFO", "DEBUG"
-compulsory=true
-
-[template variables=HOUSEKEEPING_MODE]
-ns=General
-description=How much housekeeping deletes.
-help=None will delete nothing.
+[template variables]
 
-    Debug will only delete the downloads of initial data, and will keep
-    intermediate files.
+[template variables=CSET_CYCLING_MODE]
+ns=Models and Cases
+title=Cycling mode
+description=Process case studies, or a continuous trial.
+values="case_study", "trial"
+value-titles=Case Study, Trial
+trigger=template variables=CSET_CASE_DATES: "case_study";
+        template variables=CSET_TRIAL_START_DATE: "trial";
+        template variables=CSET_TRIAL_END_DATE: "trial";
+        template variables=CSET_TRIAL_CYCLE_PERIOD: "trial";
+compulsory=true
+sort-key=a1
+
+# Case study settings.
+[template variables=CSET_CASE_DATES]
+ns=Models and Cases
+title=Case study dates
+description=List of datetimes of cases.
+help=This should be a python list of ISO 8601 datetime strings indicating the
+    forecast initiation time (AKA data time) of the data. E.g. 2000-01-01T00:00Z
```

Comment:
at the moment the data are being processed from the initialisation time. Is there already the possibility to introduce an offset? I think you have dealt with this for the sub-km models. In a further ticket do we later on need to consider cases where users subselect a datarange?

Reply:
It is, though it is currently globally applied to all the cases and models. I've had feedback that we should make it per-model.

Reply:
This is also a valid date. Should the example be updated?

Reply:
Yes please.

Reply:
Fixed in 244279e3d77577bf135bc8d8fc7d0afe21123425